### PR TITLE
Testing ABI3 wheels to reduce number of wheels

### PR DESCRIPTION
**Pull request by Narsil**

### CODE REVIEW

File: .github/workflows/python.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: windows-latest
     strategy:
       matrix:
-        python: ["3.7", "3.8", "3.9", "3.10"]
+        python: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```
Comment (ArthurZucker) [inline .github/workflows/python.yml:19]: nice thanks!

File: bindings/python/src/tokenizer.rs [hunk 2]
```diff
@@ -323,15 +323,16 @@ impl FromPyObject<'_> for PyArrayUnicode {
...
-                    let unicode = pyo3::ffi::PyUnicode_FromKindAndData(
-                        pyo3::ffi::PyUnicode_4BYTE_KIND as _,
-                        bytes.as_ptr() as *const _,
-                        elsize as isize / alignment as isize,
-                    );
-                    let py = ob.py();
-                    let obj = PyObject::from_owned_ptr(py, unicode);
-                    let s = obj.downcast_bound::<PyString>(py)?;
-                    Ok(s.to_string_lossy().trim_matches(char::from(0)).to_owned())
+                    Ok(std::str::from_utf8(bytes)?.to_owned())
```
Comment (ArthurZucker) [inline bindings/python/src/tokenizer.rs:326]: big win there

File: flake.lock
Comment (ArthurZucker) [inline flake.lock:1]: to remove

**Review by ArthurZucker — COMMENTED** [review]
LG, we can try a RC for that matter

File: flake.lock
Comment (Narsil) [reply flake.lock:1]: I understand if you wish to remove. Having those is extremely practical when using devenvs/nix. 
I'm currently keeping copies on every machine, and rewriting them all the time is just a tad annoying.

I'll remove for now, and convert you later :)

File: .github/workflows/python-release.yml [hunk 3]
```diff
@@ -164,8 +164,9 @@ jobs:
         with:
           path: ./bindings/python/dist
           merge-multiple: true
-      - name: Upload to PyPi
-        working-directory: ./bindings/python
```
Comment (ArthurZucker) [inline .github/workflows/python-release.yml:168]: do you want to do a RC for this?

**Review by ArthurZucker — APPROVED** [review]
LGTM let's go with a RC!

**Pull request merged**